util/fetch/src/client.rs

```diff
@@ -126,6 +126,10 @@ impl Client {
 		*self.client.write() = (time::Instant::now(), client.clone());
 		Ok(client)
 	}
+
+	pub fn pool(&self) -> CpuPool {
```

Comment:
If you prefer I could change it to create the pool outside `Fetch` and pass it to its constructor (and `Ntp`'s).